src/runtime.js

```diff
@@ -103,6 +105,12 @@ export function installRuntimeServices(global) {
   installTemplatesService(global);
   installTimerService(global);
   installVsyncService(global);
+  if (getMode(global).localDev || getMode(global).development ||
+      getMode(global).test || isCanary(global)) {
```

Comment:
Hey dummy, this shouldn't be served to 1% production traffic. Developer (and/or `mutate` experiment) opt-in only.